gamestonk_terminal/economy/alphavantage_model.py

```diff
@@ -0,0 +1,21 @@
+""" Alpha Vantage Model """
+__docformat__ = "numpy"
+
+from alpha_vantage.sectorperformance import SectorPerformances
+from gamestonk_terminal import config_terminal as cfg
+
+
+def get_sector_data():
+    """Get real-time performance sector data
+
+    Returns
```

Comment:
So I have been trying to mypy the returns, so `-> pd.DataFrame`.  Not sure your thoughts, but I think that helps mypy elsewhere

Reply:
Agreed. This was me forgetting!